common/src/main/java/com/microsoft/identity/common/internal/telemetry/TelemetryEventStrings.java

```diff
@@ -0,0 +1,189 @@
+// Copyright (c) Microsoft Corporation.
+// All rights reserved.
+//
+// This code is licensed under the MIT License.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files(the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and / or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions :
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+package com.microsoft.identity.common.internal.telemetry;
+
+public final class TelemetryEventStrings {
```

Comment:
One suggestion to keep this class organized - using static inner classes might force ppl to keep this organized when added new constants.

Something like....

```java
public final class TelemetryConstants {

    public static final class App {
        public static final String BUILD = "app_build";
    }

    public static final class Device {
         public static final String MANUFACTURER = "device_manufacturer";
         public static final String MODEL = "device_model";
         public static final String NAME = "device_name";
         public static final String TIMEZONE = "time_zone";
    }

    public static final class Os {
        public static final String NAME = "os_name";
    }

    public static final class Events {
        public static final String API_EVENT = "api_event";
        public static final String API_EVENT_START = API_EVENT + "_start";
        public static final String API_EVENT_END = API_EVENT + "_end";
    }

    public static final class Keys {
        public static final String EVENT_NAME = "event_name";
        public static final String EVENT_TYPE = "event_type";
    }
}
```

Everything nice and sorted, avoids the hungarian notation tarpit, works with intellisense, less typing, easy to keep from becoming a giant dumping ground of strings

Reply:
Working on this.

Reply:
Done.